packages/capacitor/package.json

```diff
@@ -24,10 +24,10 @@
   "dependencies": {
     "tslib": "^2.3.0",
     "ignore": "^5.2.4",
-    "@nx/devkit": "17.1.1",
+    "@nx/devkit": "17.1.2",
```

Comment:
For the package JSON of the package it’s better to add the Nx versions with ^. Because if you add specific versions only, the package manager complains on install that the minor version don’t fit